src/main/java/io/appium/java_client/serverevents/CommandEvent.java

```diff
@@ -0,0 +1,13 @@
+package io.appium.java_client.serverevents;
+
+public class CommandEvent {
```

Comment:
Consider using Lombok annotation for data class creation

Reply:
I initially did this, but the project wouldn't build. I did it exactly as we do with the execute driver script results, but for some reason it did not work.